pkg/network/ovn_kubernetes.go

```diff
@@ -289,6 +289,13 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 
 	data.Data["ReachabilityTotalTimeoutSeconds"] = c.EgressIPConfig.ReachabilityTotalTimeoutSeconds
 
+	reachability_node_port := os.Getenv("OVN_EGRESSIP_HEALTHCHECK_PORT")
+	if len(reachability_node_port) == 0 {
+		reachability_node_port = "9107"
```

Comment:
NIT: Please define a `const` var for the default value with a comment linking to https://github.com/openshift/enhancements/pull/1209